src/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -142,9 +154,22 @@ public void groupChat(String chatGroupName, String chatMessage, String msgSender
 				continue;
 			}
 			else{
-				String grayMessage = ChatColor.GRAY + "[" + groupName + "] " + msgSender + ": ";
-				String whiteMessage = ChatColor.WHITE + chatMessage;
-				p.sendMessage(grayMessage + whiteMessage);
+				String grayMessage = sb.append(ChatColor.GRAY)
+										.append( "[")
```

Comment:
Just my old eyes, but I suppose this is why I don't mind concatenation. Having a bunch of danglies off in the right margin somewhere makes my head hurt.

Also, [your rage is unfounded](http://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java) -- the JVM is likely just converting raw concats into the StringBuilder pattern behind the scenes, stop letting it anger you and embrace the enhanced readability.